Invalid JSON object: null

I'm in the process of upgrading to vertx 4, and in the process, I'm upgrading to the latest:

- vertx 3.9.5 -> 4.3.1
- jooq 3.13.6 -> 3.16.7
- vertx-jooq 5.2.1 -> 6.5.3

I have a (postgresql) table with a `json` column, of which much of the data is `null`. This use to work fine in the starting versions above, but now I'm running into this issue when trying to load POJOs for this table.

```
...
Caused by: io.vertx.core.json.DecodeException: Invalid JSON object: null
	at io.vertx.core.json.JsonObject.<init>(JsonObject.java:53)
	at io.github.jklingsporn.vertx.jooq.shared.postgres.JSONToJsonObjectConverter.from(JSONToJsonObjectConverter.java:21)
	at io.github.jklingsporn.vertx.jooq.shared.postgres.JSONToJsonObjectConverter.from(JSONToJsonObjectConverter.java:10)
	at org.jooq.impl.DefaultBindingGetResultSetContext$1.value(DefaultBindingGetResultSetContext.java:104)
	at io.github.jklingsporn.vertx.jooq.shared.postgres.PGJsonToVertxJsonBinding.get(PGJsonToVertxJsonBinding.java:47)
	at org.jooq.impl.CursorImpl$CursorIterator$CursorRecordInitialiser.setValue(CursorImpl.java:1578)
	... 19 common frames omitted
```

I've tried setting a default value with 
```
UPDATE halo.substation
  SET additional_configuration = to_json('{}'::text)
  WHERE additional_configuration IS NULL;
ALTER TABLE halo.substation ALTER additional_configuration 
  SET DEFAULT to_json('{}'::text);
```
but then I run into issues parsing `"{}"` (which is different than `{}`, which works fine when passed to `JsonObject`):
```
...
used by: io.vertx.core.json.DecodeException: Failed to decode:Cannot construct instance of `java.util.LinkedHashMap` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('{}')
 at [Source: (String)""{}""; line: 1, column: 1]
	at io.vertx.core.json.jackson.DatabindCodec.fromParser(DatabindCodec.java:129)
	at io.vertx.core.json.jackson.DatabindCodec.fromString(DatabindCodec.java:90)
	at io.vertx.core.json.JsonObject.fromJson(JsonObject.java:944)
	at io.vertx.core.json.JsonObject.<init>(JsonObject.java:51)
	at io.github.jklingsporn.vertx.jooq.shared.postgres.JSONToJsonObjectConverter.from(JSONToJsonObjectConverter.java:21)
	at io.github.jklingsporn.vertx.jooq.shared.postgres.JSONToJsonObjectConverter.from(JSONToJsonObjectConverter.java:10)
	at org.jooq.impl.DefaultBindingGetResultSetContext$1.value(DefaultBindingGetResultSetContext.java:104)
	at io.github.jklingsporn.vertx.jooq.shared.postgres.PGJsonToVertxJsonBinding.get(PGJsonToVertxJsonBinding.java:47)
	at org.jooq.impl.CursorImpl$CursorIterator$CursorRecordInitialiser.setValue(CursorImpl.java:1578)
	... 19 common frames omitted
Caused by: com.fasterxml.jackson.databind.exc.MismatchedInputException: Cannot construct instance of `java.util.LinkedHashMap` (although at least one Creator exists): no String-argument constructor/factory method to deserialize from String value ('{}')
 at [Source: (String)""{}""; line: 1, column: 1]
	at com.fasterxml.jackson.databind.exc.MismatchedInputException.from(MismatchedInputException.java:63)
	at com.fasterxml.jackson.databind.DeserializationContext.reportInputMismatch(DeserializationContext.java:1728)
	at com.fasterxml.jackson.databind.DeserializationContext.handleMissingInstantiator(DeserializationContext.java:1353)
	at com.fasterxml.jackson.databind.deser.std.StdDeserializer._deserializeFromString(StdDeserializer.java:311)
	at com.fasterxml.jackson.databind.deser.std.MapDeserializer.deserialize(MapDeserializer.java:444)
	at com.fasterxml.jackson.databind.deser.std.MapDeserializer.deserialize(MapDeserializer.java:32)
	at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:323)
	at com.fasterxml.jackson.databind.ObjectMapper._readValue(ObjectMapper.java:4650)
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:2831)
	at io.vertx.core.json.jackson.DatabindCodec.fromParser(DatabindCodec.java:126)
	... 27 common frames omitted
 ```


I'm not sure if vertx changed to no longer allow a null value in the constructor of `JsonObject` or if something needs to change in the `JSONtoJsonObjectConverter`.  If it's the later, I'm happy to submit a PR to change

--- a/vertx-jooq-shared/src/main/java/io/github/jklingsporn/vertx/jooq/shared/postgres/JSONToJsonObjectConverter.java
+++ b/vertx-jooq-shared/src/main/java/io/github/jklingsporn/vertx/jooq/shared/postgres/JSONToJsonObjectConverter.java
@@ -18,7 +18,7 @@ public class JSONToJsonObjectConverter implements PgConverter<JsonObject, JSON, 
 
     @Override
     public JsonObject from(JSON t) {
-        return t == null || t.data() == null ? null : new JsonObject(t.data());
+        return t == null || t.data() == null || t.data().equals("null") ? null : new JsonObject(t.data());
     }
 
     @Override

